search.js

```diff
@@ -48,8 +46,15 @@ const askForSearch = () => {
       )
       console.log(`${index + 1}: ${coloredPath}`)
     })
-    askForKara(karaFound).then(askForSearch)
+    askForKara(karaFound).then(() => askForSearch(allContents))
   })
 }
 
-askForSearch()
+process.stdout.write('Loading data...')
```

Comment:
console.log automatically append a \n at the end of the message.
This method allows me to use the \r to erase the previous line.
So "loading data" is an ephemerous message